fixubuntu.sh

```diff
@@ -38,7 +38,7 @@ fi
 $GS set "$CCUL" remote-content-search none
 
 # If you're using a version earlier than v13.10, uninstall unity-lens-shopping
-if [[ $V < 13.10 ]]; then
+if [[ $(echo $V'<'13.10 | bc) ]]; then
```

Comment:
This won't work. If `$V` is less than `13.10`, bc echoes `1`, otherwise echoes `0`. Then you pass that `0` or `1` to `[[`, which in that form will return 0 if its argument is empty, 1 otherwise. The string `0` and `1` are never empty strings, so in this case `[[` always returns 0 and the condition for that `if` is always true.

You can fix it by changing it to that:

    if [[ $(echo $V'<'13.10 | bc) == 1 ]]; then